src/planscape/collaboration/migrations/0002_add_permissions.py

```diff
@@ -0,0 +1,39 @@
+from django.db import migrations
+
+
+# Add the initial data to define each role and corresponding permissions
+class Migration(migrations.Migration):
+    dependencies = [
+        ("collaboration", "0001_initial"),
+    ]
+
+    def insertData(apps, schema_editor):
```

Comment:
Here im manually adding permissions to the table, not sure if this is ok or somehow we wanna reference the `auth_permission` records instead

Reply:
I've still been trying to figure that out, too :D Maybe we can spend a little time weighing the pros and cons, or if this table is useful for us at all